convertNetworkVersion/convert-network-version.go

```diff
@@ -0,0 +1,46 @@
+package convertNetworkVersion
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"os"
+
+	"github.com/mr-tron/base58/base58"
+)
+
+func ConvertNetworkVersion(originalAddress string, networkVersion byte) string {
+	decoded, err := base58.Decode(originalAddress)
```

Comment:
It doesn't look like `networkVersion` is used?

Reply:
Sloppy mistake.  I added a test to make sure this is covered correctly.